composer.json

```diff
@@ -21,19 +21,19 @@
     ],
     "require": {
         "php": "^7.1",
-        "sonata-project/admin-bundle": "^3.4",
+        "sonata-project/admin-bundle": "^3.30",
         "sonata-project/block-bundle": "^3.11",
         "sonata-project/cache-bundle": "^2.1.7",
-        "sonata-project/core-bundle": "^3.0",
-        "sonata-project/datagrid-bundle": "^2.2",
+        "sonata-project/core-bundle": "^3.9",
+        "sonata-project/datagrid-bundle": "^2.3",
         "sonata-project/doctrine-extensions": "^1.0",
-        "sonata-project/easy-extends-bundle": "^2.2",
-        "sonata-project/notification-bundle": "^3.0",
-        "symfony/config": "^2.8 || ^3.2 || ^4.0",
-        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
-        "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
-        "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
-        "symfony/security-core": "^2.8 || ^3.2 || ^4.0"
+        "sonata-project/easy-extends-bundle": "^2.3",
+        "sonata-project/notification-bundle": "^3.3",
+        "symfony/config": "^3.4 || ^4.0",
```

Comment:
Travis will complain about this changes. Maybe revert this part on a separate PR with previous dev-kit change